.github/workflows/docker-image.yml

```
@@ -0,0 +1,29 @@
+      uses: docker/setup-qemu-action@v3
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+    - name: Build and push the Docker image
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        echo "Building Docker image..."
+        REPO_NAME_LOWER=$(echo "${{ github.repository }}" | tr '[:upper:]' '[:lower:]')
+        docker buildx build --platform linux/amd64,linux/arm64 . --file Dockerfile --tag ghcr.io/$REPO_NAME_LOWER:latest --push
+        echo "Logging in to GitHub Packages..."
+        echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u ${{ github.actor }} --password-stdin
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)